test/proto/cluster_factory_config.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
```

Comment:
I know we have a test/proto filfer, but my preference would be for you to put this file next to your test cluster in that folder if you don't mind.